packages/legend-application-studio/src/components/editor/editor-group/mapping-editor/InstanceSetImplementationEditor.tsx

```diff
@@ -101,6 +101,8 @@ export const InstanceSetImplementationSourceExplorer = observer(
     const { setImplementation, isReadOnly } = props;
     const editorStore = useEditorStore();
     const applicationStore = useApplicationStore();
+    const [relationalMappingStrictMode, setRelationalMappingStrictMode] =
```

Comment:
dont think this naming or the fact we reference relational in the instance set implementation. 
Could we think of another way to do this please. 
Maybe have a filter function here with the Mapping Source as input ?

Reply:
<img width="585" alt="Screenshot 2023-08-24 at 10 33 03 AM" src="https://github.com/finos/legend-studio/assets/73408381/989d86aa-51d4-4c03-be84-f983ec792104">
As the screenshot shows, this flag is used when composing the dropdown. The flag is only set to true when the source is of type database

Reply:
I updated to code to pass in the sourceElement which could be used to compose the dropdown